module3/src/controllers/user.controller.ts

```diff
@@ -0,0 +1,84 @@
+import express, { Router } from 'express';
+import { Error } from 'sequelize';
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore
+import UserService from '../services/user.service.ts';
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore
+import { validateSchema } from '../utils/utils.ts';
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore
+import { userSchema } from '../validators/user.ts';
+
+const userService = UserService();
+
+const usersRouter = Router();
+usersRouter.use(express.json());
+const userRouter = Router();
```

Comment:
duplicate code

Reply:
I did not fully understand what in this case a duplicate was.
Can you please clarify?

Reply:
My bad it's ok